content/en/synthetics/guide/_index.md

```diff
@@ -2,12 +2,14 @@
 title: Synthetic Monitoring Guides
 kind: guide
 private: true
+disable_sidebar: true
 ---
 
 {{< whatsnext desc="General Guides:" >}}
     {{< nextlink href="synthetics/guide/uptime-percentage-widget" >}}Website uptime monitoring with SLOs{{< /nextlink >}}
     {{< nextlink href="synthetics/guide/identify_synthetics_bots/" >}}Identify Synthetic bots{{< /nextlink >}}
     {{< nextlink href="synthetics/guide/monitor-usage/" >}}Monitor Synthetics usage{{< /nextlink >}}
+    {{< nextlink href="synthetics/guide/synthetic-test-monitors/" >}}Understanding Synthetic tests and Alerting monitors{{< /nextlink >}}
```

Comment:
What do you think about linking this guide from https://docs.datadoghq.com/monitors/guide/ as well?